README.md

```diff
@@ -11,7 +11,7 @@
 | Dependency | Mandatory | Notes |
 |------------|:---------:|-------|
 | [ModLib](https://www.nexusmods.com/mountandblade2bannerlord/mods/592) | ✓ | Used to provide settings and other goodies |
-| [Save Missing Module Fix](https://www.nexusmods.com/mountandblade2bannerlord/mods/282) | ✗ | Recommended as it ensures saves can be loaded when you unload this mod having saved a game with it enabled |
+| [ALTERNATIVE Save Missing Module Fix](https://github.com/bmountney/Aragas.AltSaveSystemFix/releases/tag/1.3.0.0) | ✗ | Recommended as it ensures saves can be loaded when you unload this mod having saved a game with it enabled. Tested as working with this mod (Note: Original version does not work correctly) |
```

Comment:
As far as I understood the change this mod shouldn't be listed here at all anymore. Only reason would be for older versions but then it should be stated more clearly.